packages/augur-sdk-lite/src/zeroX.ts

```diff
@@ -1,6 +1,8 @@
 import { BigNumber as BN, defaultAbiCoder, ParamType } from 'ethers/utils';
 import { getAddress } from 'ethers/utils/address';
 import { BigNumber } from 'bignumber.js';
+import { OrderTypeOrders, OrderType, ZeroXOrder, ZeroXOrders, OutcomeOrders } from '.';
+import * as _ from 'lodash';
```

Comment:
`lodash` isn't in the dependencies for SDK lite, and it would be nice to not need it.